Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift

```diff
@@ -142,6 +142,72 @@ class CLDTransformationExpressionsTests: BaseTestCase {
         XCTAssertEqual(actualResult, expectedResult, "Calling get x should return its value")
     }
     
+    func test_setStartOffset_emptyInputParamaters_shouldNotStoreNewVariable() {
+        
+        // Given
+        let input       = String()
+        let expression  = CLDExpression(value: input)
+        
+        // When
+        sut.setStartOffset(expression)
+        
+        let actualResult = sut.startOffset!
+        
+        // Then
+        XCTAssertTrue(actualResult.isEmpty, "Empty expression should not be stored in params")
+    }
+    
+    func test_setStartOffset_inputExpression_shouldStoreNewValue() {
+        
+        // Given
+        let input       = "initialHeight * 2"
```

Comment:
The value of the expression does not represent a real world use case, like `initialHeight` is not related to video offset... Please use more related variables, like `duration`. The test content is very confusing.

Reply:
Fixed